integration_tests/integration_test.go

```diff
@@ -455,6 +452,69 @@ var _ = Describe("LiveIntegrationTests", func() {
 
 	//})
 
+	Describe("cloud storage", func() {
+		It("can provision/bind/unbind/deprovision", func() {
+			service, err := googlestorage.NewClient(context.Background(), option.WithUserAgent(models.CustomUserAgent))
+			Expect(err).NotTo(HaveOccurred())
+
+			bucketName := "integration_test_bucket"
+			params := &genericService{
+				serviceId:          serviceNameToId[brokers.StorageName],
+				planId:             serviceNameToPlanId[brokers.StorageName],
+				instanceId:         bucketName,
+				bindingId:          "integration_test_bucket_binding",
+				rawProvisionParams: []byte("{\"name\": \"" + bucketName + "\"}"),
+				rawBindingParams: map[string]interface{}{
+					"role": "editor",
+				},
+				serviceExistsFn: func(bool) bool {
+					bucket := service.Bucket(bucketName)
+					_, err = bucket.List(context.Background(), nil)
```

Comment:
I've changed this to `bucket.Attrs(..)` which will retrun an err if it doesn't exist. This looks to be the equivalent of calling [Get](https://cloud.google.com/storage/docs/json_api/v1/buckets/get) but via the BucketHandle.